client/src/components/ProductOverview/ProductInfo.jsx

```diff
@@ -1,11 +1,16 @@
 import React from 'react';
 import s from './style.css';
 
-const ProductInfo = () => {
-
+const ProductInfo = ({ productInfo }) => {
   return (
     <div id={s.info}>
-      <h1>Product Info</h1>
+      <p>
+        <span>&#9733;&#9733;&#9733;&#9733;&#9733; </span>
```

Comment:
Better use Star components to show average star ratings!, and if hardcode is needed, you can copy and paste star unicode directly.